openstudiocore/src/utilities/bcl/LocalBCL.cpp

```diff
@@ -103,8 +111,8 @@ namespace openstudio{
     if (!ptr) {
       QSettings settings("OpenStudio", "LocalBCL");
```

Comment:
@lefticus got any ideas for what to replace QSettings with?

Reply:
We could try to roll something around boost::property_tree or maybe one of the existing JSON libraries used at NREL? I don't have any clear idea.